### PR TITLE
Update CoRIM version to the latest git revision

### PR DESCRIPTION
**Pull request by jpbrucker**
There were several breaking changes to the API:

* comid.NewInstance() takes two parameters
* AttestVerifKey and ReferenceValue replaced by ValueTriple
* comid.NewCCAPlatCfgMeasurement and comid.NewPSAMeasurement return an additional error variable

And more importantly, there now is a single measurement per reference value. Replace GetMeasurementsFromComponents() with GetRefValsFromComponents(), which returns the new reference values list instead of measurements.

This fixes issue https://github.com/veraison/gen-corim/issues/5

### CODE REVIEW

File: cmd/gen-corim.go [hunk 1]
```diff
@@ -141,39 +141,40 @@ func generate(attestation_scheme *string, evidence_file *string, key_file *strin
...
-	//creating a new reference value containing the measurements and the implementation ID from the evidence token
+	//creating new reference values containing the measurements and the implementation ID from the evidence token
 	class := comid.NewClassImplID(schemeClaims.implID)
 
-	refVal := comid.ReferenceValue{
-		Environment:  comid.Environment{Class: class},
-		Measurements: measurements,
+	refVals, err := GetRefValsFromComponents(schemeClaims, class, *attestation_scheme == "cca")
+	if err != nil {
+		_ = os.RemoveAll(dir)
```
Comment (thomas-fossati) [inline cmd/gen-corim.go:149]: Just a note: this (and all the similar instances above and below) could be handled with a single [`defer`](https://go.dev/tour/flowcontrol/12) statement.

**Review by thomas-fossati — APPROVED** [review]
LGTM - eye testing only ;-)

Thanks a lot!

**Comment by yogeshbdeshpande** [comment]
Thank you for the fix. Nice Work!

As we do not have any sort of CI tests kindly test that the new version of tool (which has your PR) constructs the CoRIM correctly.

Longer term, we need to fix the pipeline also, but that can be a separate issue to track!

**Review by yogeshbdeshpande — APPROVED** [review]
LGTM!

**Comment by jpbrucker** [comment]
> Thank you for the fix. Nice Work!
> 
> As we do not have any sort of CI tests kindly test that the new version of tool (which has your PR) constructs the CoRIM correctly.
> 
> Longer term, we need to fix the pipeline also, but that can be a separate issue to track!

Thanks for the review! I tested the changes by provisioning veraison with the output file and performing a remote attestation

**Comment by yogeshbdeshpande** [comment]
Awesome work!

**Pull request merged**